main.go

```diff
@@ -18,6 +18,9 @@ import (
 // Global for Debug logging flag.
 var debug bool
 
+// Global for version string.
+var version string
```

Comment:
nit: can we combine the vars?

Reply:
No problem - that'd be my go ignorance on display :)